x-pack/metricbeat/module/stan/stats/stats.go

```diff
@@ -0,0 +1,79 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package stats
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/metricbeat/helper"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+)
+
+const (
+	defaultScheme = "http"
+	defaultPath   = "/streaming/serverz"
```

Comment:
`/serverz/` looks weird. Is it correct path?

Reply:
🙂 Yeap, those `Zs` is a "nats" thing. https://github.com/nats-io/nats-streaming-server/blob/b95714efeef2e1bcc46d11da6f41bc88da1e1f04/server/monitor.go#L35